File: Program5.c
 //Problem Statement : Accept two values from user and perform the Addition
 //Conclusion :Additon is a function which accepts two parameter as integer and it returns integer

// Step 1: Understand the problem statemnt 


 // Step2 :Write the Algorithm
 // Accept first value from user  store it in iNo1
 //Accept second value from user and store it into iNo2
 //Crete third Variable to store result 

 //Step3 :  Select the programing language(c, c++ ,java ,python)

 //step 4 :Write program


#include<stdio.h>


int Addition(int iNo1 ,int iNo2)

{
  int iSum = 0;
  iSum = iNo1 + iNo2;
  return iSum;

 
}

int main()
{

    int iValue1 = 0;
    int iValue2 = 0;
    int iAns = 0;
    


   printf("Enter first number :\n");
   scanf("%d",&iValue1);

   printf("Enter first number :\n");
   scanf("%d",&iValue2);

   iAns = Addition(iValue1 , iValue2);

   printf("Additon is : %d\n",iAns);


    return 0;
    
}

//Step 5 : Test the code

/*
Test Case 1 ;
Input : 10 11
Output : 21

Test Case 2 :
Input :12 -6
Output : 6

Test Case 4 :
Input : -6 -5
Output : -11

*/

File: Java Programs/Problems on Numbers/Program5.java

//Accept two numbers from user and perform addition of that numbers display result

import java.util.*;


class Program2
{
    public static int Addition(int iNo1, int iNo2)
    {
            int iResult = 0;
            iResult = iNo1 + iNo2;
            
            return iResult;
    }

    public static void main(String Args[])
    {

        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0;
        int iValue2 = 0;
        int iRet = 0;
         
        System.out.println("Enter First Number :");
        iValue1 = sobj.nextInt();

        System.out.println("Enter Second Number :");
        iValue2 = sobj.nextInt();

        iRet = Addition(iValue1, iValue2); //Fucntion call
        
        System.out.println("Addition of Two Numbers :"+iRet);


    }
}